src/patternfly/_chart-globals.scss

```diff
@@ -50,3 +51,189 @@ $pf-chart-color-black-200: $pf-color-black-400;
+$pf-chart-axis--grid--PointerEvents: "painted";
+$pf-chart-axis--tick--fill: transparent;
+$pf-chart-axis--tick--Width: 1;
+$pf-chart-axis--tick--stroke--Color: transparent;
+$pf-chart-axis--tick-label--fill: $pf-chart-color-black-500;
+
+// Bar Chart
+$pf-chart-bar--Width: 10;
+$pf-chart-bar--data--stroke: none;
+$pf-chart-bar--data--fill: $pf-color-black-900;
+$pf-chart-bar--data--Padding: 8;
+$pf-chart-bar--data-stroke--Width: 0;
+
+// Box Plot Chart
+$pf-chart-boxplot--max--Padding: 8;
+$pf-chart-boxplot--max--stroke--Color: $pf-color-black-900;
+$pf-chart-boxplot--max--stroke--Width: 1;
+$pf-chart-boxplot--median--Padding: 8;
+$pf-chart-boxplot--median--stroke--Color: $pf-color-black-900;
+$pf-chart-boxplot--median--stroke--Width: 1;
+$pf-chart-boxplot--min--Padding: 8;
+$pf-chart-boxplot--min--stroke--Width: 1;
+$pf-chart-boxplot--min--stroke--Color: $pf-color-black-900;
+$pf-chart-boxplot--lower-quartile--Padding: 8;
+$pf-chart-boxplot--lower-quartile--fill: $pf-chart-color-black-300;
+$pf-chart-boxplot--upper-quartile--Padding: 8;
+$pf-chart-boxplot--upper-quartile--fill: $pf-chart-color-black-300;
+$pf-chart-boxplot--box--Width: 20;
+
+// Bullet Chart
+$pf-chart-bullet--Height: 55;
+$pf-chart-bullet--items-per-row: 2;
+$pf-chart-bullet--legend--Height: 30;
+$pf-chart-bullet--range--Width: 40;
+$pf-chart-bullet--threshold-error--Color: $pf-color-red-100;
+$pf-chart-bullet--threshold-error--Width: 2;
+$pf-chart-bullet--value--Width: 9;
+$pf-chart-bullet--comparative-measure-line--warning--stroke--Color: $pf-color-orange-300;
+$pf-chart-bullet--comparative-measure-line--warning--BorderWidth: 1;
+$pf-chart-bullet--comparative-measure-line--error--stroke--Color: $pf-color-red-100;
+$pf-chart-bullet--comparative-measure-line--error--BorderWidth: 1;
+$pf-chart-bullet--zero-line--stroke--Color: $pf-color-black-700;
+$pf-chart-bullet--zero-line--BorderWidth: 1;
+$pf-chart-bullet--negative-primary-measure--stroke--Color: $pf-color-red-100;
+$pf-chart-bullet--negative-primary-measure--BorderWidth: 2;
+
+// Candlestick
+$pf-chart-candelstick--data--stroke--Width: 1;
+$pf-chart-candelstick--data--stroke--Color: $pf-color-black-900;
+$pf-chart-candelstick--candle--positive--Color: $pf-color-white;
+$pf-chart-candelstick--candle--negative--Color: $pf-color-black-900;
+
+// Chart
+$pf-chart-chart--BorderWidth: 1 solid;
+
+// Simple Donut Chart
+$pf-chart-donut--label--subtitle--fill: $pf-chart-color-black-200;
+$pf-chart-donut--pie--Height: 230;
+$pf-chart-donut--pie--angle--Padding: 1;
+$pf-chart-donut--pie--Width: 230;
+
+// Donut Threshold Chart
+$pf-chart-donut--threshold--first--Color: $pf-color-black-200;
+$pf-chart-donut--threshold--second--Color: $pf-color-black-300;
+$pf-chart-donut--threshold--third--Color: $pf-color-black-400;
+$pf-chart-donut--threshold--success--Color: $pf-chart-color-blue-300;
+$pf-chart-donut--threshold--warning--Color: $pf-chart-color-gold-400;
+$pf-chart-donut--threshold--danger--Color: $pf-color-red-100;
```

Comment:
for vars like this where we reference the PF4 color var directly, I'm curious if we should...

1) add this as another `$pf-chart-color-red-[...]` color?
2) add another layer of meaningful color var names, like we have in core, for `$pf-chart-global--danger-color--100` and use it where we use "danger" colors?
3) if we don't want to do that, would it be better to reference `$pf-global--danger-color--100` instead of `$pf-color-red-100` so that this danger chart color always aligns with our primary danger color in core?

Reply:
spoke with @gdoyle1 and she said it would be good to replace those red colors with `$pf-global--danger-color--100` and `pf-color-black-900` with the global text color. Updated

Reply:
I believe these particular vars will be specific to the donut utilization chart. 

We could reference global colors in react, but I suspect this will make it easier to override just the donut utilization chart?

Reply:
I believe these particular vars will be specific to the donut utilization chart. For example, the bullet chart has different color thresholds for warning and error.

We could reference global colors in react, but I suspect this will help make it easier to override?